.prettierrc.json

```diff
@@ -2,5 +2,6 @@
   "semi": false,
   "singleQuote": true,
   "jsxSingleQuote": true,
+  "trailingComma": "all",
```

Comment:
This is the new prettier v3 default, but it has to be set here because the icon generator uses the old default.